css/DayPicker.scss

```diff
@@ -29,7 +29,9 @@
   color: $react-dates-color-placeholder-text;
   cursor: pointer;
   line-height: 0.78;
-  user-select: none;
+  -webkit-user-select: none; /* Chrome/Safari */
```

Comment:
This removes the standard one in favor the vendor prefixes, which ideally should be handled by something like autoprefixer. At the very least, we must keep the standard one.